### PR TITLE
Add project permissions component and filter dropdown by env

### PR DESCRIPTION
**Pull request by drshrey**

### CODE REVIEW

**Comment by drshrey** [comment]
screenshot 2018-03-14 00 44 21

**Comment by drshrey** [comment]
screenshot 2018-03-14 00 50 14

File: src/components/Project/Settings/index.js [hunk 1]
```diff
@@ -91,6 +115,7 @@ export default class Settings extends React.Component {
   }
 
   updateSettings(form){
+    console.log(form.values())
```
Comment (sasso) [inline src/components/Project/Settings/index.js]: X

File: src/components/App/index.js [hunk 2]
```diff
@@ -112,11 +112,18 @@ export default class App extends React.Component {
       if(!user){
           return <Redirect to={{pathname: '/login', state: { from: this.props.location }}}/>
       }
+
+      // parse slug from window.location since props.match is empty
+      let slug = ""
+      if(this.props.location.pathname.includes("projects")){
+        slug = this.props.location.pathname.split("/")[2]
```
Comment (sasso) [inline src/components/App/index.js]: Is there no other way to get this from the router in a sub component?

**Pull request merged**